Hero Patch for flex-direction vis diff items. (#620)
microsoft/atlas-design: .changeset/new-pets-speak.md

---
'@microsoft/atlas-css': patch
---

> Remove explicit flex-direction from the hero component in favor of atomics.

Reviewer: Let's use past tense here too.

> Removing the flex-direction from the hero component and will apply the direction manually via templates and scripts

Reviewer 2: Given the function of this message here is to populate our changelog, what would you say to something like this?

```suggestion
Remove explicit flex-direction from the hero component in favor of atomics.
```

Author: should be updated now.
